site/docs/v1/tech/identity-providers/twitter.adoc

```diff
@@ -37,40 +37,35 @@ Once you have logged into Twitter and have obtained access to a Twitter develope
 
-* https://developer.twitter.com/en/apps
+* https://developer.twitter.com/en/portal/projects-and-apps
 
-In the following screenshot you will find an example of the Create an App process. Review the following sections to ensure your configuration is correct. Fill out any required fields found on this form and click the `Create` button to create a new Twitter app.
+In the following screenshot you will find an example of the Create an App process. Review the following sections to ensure your configuration is correct.
 
-=== Enable Sign in with Twitter
+=== Record the Keys and Token
 
-Ensure that you check `Enable Sign in with Twitter` checkbox. If this is not checked the Twitter login will not function properly in FusionAuth.
+The API key and API secret key are typically presented at the onset of app creation. Record these values in a password vault (or similar) for safekeeping. They will be needed for later integration with the FusionAuth IdP.
 
-=== Add the Callback URL
+=== Authentication Settings
 
-In this example I have registered `\https://login.piedpiper.com/oauth2/callback` as the callback URL.
+You will need to modify a few setting here to enable Twitter to connect with FusionAuth.
 
-The first part of that value will be equal to the public URL of FusionAuth, in this case `\https://login.piedpiper.com`. The second part, the actual redirect URI will always be `/oauth2/callback`.
-
-image::identity-providers/twitter-create-app.png[Create Twitter APp,width=1200,role=shadowed]
-
-
-=== Configure Permissions
+==== Add the Callback URL +++<span style"height: 50%; width: 50%;"><span class="fa-stack fa-2x" style="color: #F4821F;"><i class="fa fa-circle fa-stack-1x"></i><span class="fa-stack-1x calendar-text" style="color: black; font-family: sans-serif; font-size: medium">1</span></span>+++ & +++<span class="fa-stack fa-2x" style="color: #F4821F;"><i class="fa fa-circle fa-stack-1x"></i><span class="fa-stack-1x calendar-text" style="color: black; font-family: sans-serif; font-size: medium">2</span></span>+++
 
-Now you should have a Twitter app created, select the `Permissions` tab and click the `Edit` button in the top right of the panel.
+In this example I have registered `\https://login.piedpiper.com/oauth2/callback` as the callback URL. Please note that you will need to `enable` 3-legged OAuth and complete all required fields.
 
-You may select `Read-only` in the `Access permission` section and under `Additional permissions` ensure you have checked `Request email address from users`. An email address is required to complete login in FusionAuth.
+The first part of that value will be equal to the public URL of FusionAuth, in this case `\https://login.piedpiper.com`. The second part, the actual redirect URI will always be `/oauth2/callback`.
```

Comment:
Which value? This is confusing.

`/oauth2/callback` isn't a URI (no hostname, no scheme). So I'm confused.